### PR TITLE
Add tos attributes to onboarding and rename Exit event

### PR DESCRIPTION
**Pull request by jojo-stripe**

### CODE REVIEW

File: src/Components.tsx [hunk 1]
```diff
@@ -32,18 +32,29 @@ export const ConnectPaymentDetails = ({
 };
 
 export const ConnectAccountOnboarding = ({
-  onOnboardingExited,
+  onExit,
+  recipientTermsOfServiceUrl,
+  fullTermsOfServiceUrl,
 }: {
-  onOnboardingExited: () => void;
+  onExit: () => void;
```
Comment (jorgea-stripe) [inline src/Components.tsx:39]: Let's make sure we don't deploy the `onExit` change before that change to `submerchant-surfaces` is deployed.

File: src/Components.tsx [hunk 2]
```diff
@@ -32,18 +32,29 @@ export const ConnectPaymentDetails = ({
...
+  recipientTermsOfServiceUrl?: string;
+  fullTermsOfServiceUrl?: string;
 }): JSX.Element | null => {
   const {wrapper, component: onboarding} =
     useCreateComponent('account-onboarding');
 
-  useAttachEvent(
+  useAttachAttribute(
```
Comment (jorgea-stripe) [inline src/Components.tsx:46]: We are missing a few attributes here! See our API review doc for the full list

File: src/Components.tsx (hunk 1 above)
Comment (nathanbullenmyob) [reply src/Components.tsx:39]: Hey @jorgea-stripe / @jojo-stripe 

We currently listen for `onOnboardingExited`. When the service is deployed to change to `onExit`, will our code break even if we don't upgrade our `react-connect-js` version?

Comment (jojo-stripe) [reply src/Components.tsx:39]: No, we will be emitting both events for backwards compatibility!

Comment (nathanbullenmyob) [reply src/Components.tsx:39]: Beautiful! 😄 Thanks for the reply

**Review by jorgea-stripe — APPROVED** [review]

**Pull request merged**